src/Helpers/CheckoutHelper.php

```diff
@@ -1096,6 +1096,20 @@ public function refundTransaction($order, $context, $item, $state, &$orderItems
         $customFields['serviceName']            = $item['transaction_method'];
         $customFields['originalTransactionKey'] = $item['transactions'];
 
+        $cardTypes = [
+            'mastercard',
+            'visa',
+            'amex',
+            'vpay',
+            'maestro',
+            'visaelectron',
+            'cartebleuevisa',
+            'cartebancaire',
+            'dankort',
+            'nexi',
+            'postepay'
+        ];
+
         $serviceName = (in_array($customFields['serviceName'], ['creditcard','creditcards', 'giftcards'])) ? $customFields['brqPaymentMethod'] : $customFields['serviceName'];
```

Comment:
why not just remove 'creditcard' from here?

Reply:
what would you achieve from that, the problem is that  $customFields['serviceName'] is for example visa and not creditcard

Reply:
sorry about that, your solution is technically ok, but what about giftcards and the redirect credicards, isn't this bug happening there as well?

Reply:
yeah you are right, it was happening, fixed in current update